src/utilities/profiles/ProfileBuilder.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  ActionRowBuilder,
+  ButtonBuilder,
+  ButtonStyle,
+  EmbedBuilder,
+  SelectMenuInteraction,
+  StringSelectMenuInteraction,
+} from "discord.js";
+import { ProfileOutput } from "../../models/Profile";
+import render, { getColor } from "./render";
+import Badges from "./badges";
+import Query from "../../routes/Query";
+import { CommentOutput } from "../../models/Comment";
+
+export default class ProfileBuilder {
+  static async renderHome(profile: ProfileOutput): Promise<EmbedBuilder> {
+    return await render(profile);
+  }
+
+  static async renderBadges(profile: ProfileOutput): Promise<EmbedBuilder> {
+    let badges = Badges.getBadges(profile.badge_flags);
+    let desc = Badges.getDescriptions(profile.badge_flags).join("\n");
+    if (desc.length == 0) desc = "This user has no badges.";
+    return new EmbedBuilder()
+      .setTitle(`${profile.display_name}'s Badges`)
+      .setColor(getColor(badges.names))
+      .setDescription(`${desc}`);
+  }
```

Comment:
Avoid abbreviations.
```suggestion
    let badges = Badges.getBadges(profile.badge_flags);
    let description = Badges.getDescriptions(profile.badge_flags).join("\n");
    if (description == 0) desc = "This user has no badges.";
    return new EmbedBuilder()
      .setTitle(`${profile.display_name}'s Badges`)
      .setColor(getColor(badges.names))
      .setDescription(`${description}`);
  }
```